sync-core/src/main/java/com/cloudant/sync/replication/PullReplication.java

```diff
@@ -54,6 +56,28 @@ public PullReplication() {
         /* Does nothing but we can now document it */
     }
 
+
```

Comment:
Delete these extra newlines - we should normally have one blank line between methods